src/design-tokens/mixins.css

```diff
@@ -65,24 +65,21 @@
 }
 
 @define-mixin focus {
-  outline: var(--eds-border-width-sm) solid var(--eds-theme-color-focus-ring);
-  outline-offset: 2px;
+  outline: var(--eds-theme-box-focus-ring-outline-width) solid
+    var(--eds-theme-color-focus-ring);
+  outline-offset: var(--eds-theme-box-focus-ring-outline-offset);
 }
 
 @define-mixin focusInside {
   @mixin focus;
-  outline-offset: -2px;
+  outline-offset: calc(-1 * var(--eds-theme-box-focus-ring-outline-offset));
 }
 
 @define-mixin focusInverted {
   @mixin focus;
   outline-color: var(--eds-theme-color-focus-ring-inverted);
 }
 
-@define-mixin focusTransparent {
```

Comment:
++

Reply:
Yea removed this one since it wasn't being used anywhere idk why we had this

Reply:
if focus were implemented with borders or something that changes layout, you'd want a transparent in place, so that stuff doesn't slip around (and cause a dom repaint). it's possible an older version of this used to not have `outline` but they didn't clean that stuff out.